### ISSUE TITLE
Stop Playback/Fullscreen like Youtube

### ISSUE THREAD

**Issue by platinum55** [jellyfin/jellyfin#2027]
currently when doubleclicking the movie, for fullscreen, it stops for a moment... youtube handles it likes this:

when a click is recognized, it waits for 500ms - if no other click followed, then the playback stops. if a second click follows within 500ms, then fullscreen is triggered.

would be nice if someone could implement this behavior into jellyfin

**Comment by thomasshera** [comment]
I think this is a feature request not a bug

**Comment by platinum55** [comment]
indeed, but feature request is not choosable when creating an issue here anymore
